src/template.html

```diff
@@ -7,6 +7,8 @@
 
 	%sapper.base%
 
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
```

Comment:
I'm not familiar with Svelte/Sapper. Do you know whether it supports bundling CSS stylesheets? We'd like to bundle these stylesheets (as well as our custom styles) directly as one CSS file, so that these external dependency is not needed.

Reply:
Yes it does support bundling CSS files. I will make this change immediately.

Reply:
It is OK to reference external third CSS files. I am not sure how are we going to use the grid CSS framework. If the grid CSS framework is written in SCSS mixin, then we are going to bundle our CSS. If the grid CSS framework is just a CSS stylesheet, it is fine to reference it with `<link>`. Anyway, we need to bundle SCSS/CSS.

Reply:
Turns out the CSS for bootstrap grid has its own CSS file. I've included bundled it with the normalize.css in my latest commit.